docker/aeternity-dev.yaml

```diff
@@ -34,7 +34,7 @@ system:
   # plugin_path: /home/aeternity/node/plugins
   dev_mode: true
   plugins:
-    - name: ae_plugin_dev_mode
+    - name: aeplugin_dev_mode
```

Comment:
https://github.com/aeternity/aeplugin_dev_mode?tab=readme-ov-file#configuration
I'm not sure "ae_plugin_dev_mode" where comes from